go.work

```diff
@@ -27,3 +27,5 @@ replace github.com/getkin/kin-openapi => github.com/getkin/kin-openapi v0.125.0
 replace github.com/prometheus/alertmanager => github.com/grafana/prometheus-alertmanager v0.25.1-0.20240930132144-b5e64e81e8d3
 
 replace github.com/crewjam/saml => github.com/grafana/saml v0.4.15-0.20240917091248-ae3bbdad8a56
+
+replace github.com/goccy/go-json => github.com/grafana/go-json v0.0.0-20240212174302-e53f8071f0c1
```

Comment:
Can you rebase the fork so it layers on top of https://github.com/goccy/go-json/commit/5e2ae3f23c1db71990844a230a4d11559efe128e (the v0.10.3 release that we're using currently)?